analysis/create_project_actions.R

```diff
@@ -356,7 +356,7 @@ apply_stata_model_function <- function(name, cohort, analysis, ipw, strata,
   splice(
     action(
       name = glue("ready-{name}"),
-      run = glue("cox-ipw:v0.0.31 --df_input=model_input-{name}.rds --ipw={ipw} --exposure=exp_date --outcome=out_date --strata={strata} --covariate_sex={covariate_sex} --covariate_age={covariate_age} --covariate_other={covariate_other} --cox_start={cox_start} --cox_stop={cox_stop} --study_start={study_start} --study_stop={study_stop} --cut_points={cut_points} --controls_per_case={controls_per_case} --total_event_threshold={total_event_threshold} --episode_event_threshold={episode_event_threshold} --covariate_threshold={covariate_threshold} --age_spline={age_spline} --save_analysis_ready=TRUE --run_analysis=FALSE --df_output=model_output-{name}.csv"),
+      run = glue("cox-ipw:v0.0.30 --df_input=model_input-{name}.rds --ipw={ipw} --exposure=exp_date --outcome=out_date --strata={strata} --covariate_sex={covariate_sex} --covariate_age={covariate_age} --covariate_other={covariate_other} --cox_start={cox_start} --cox_stop={cox_stop} --study_start={study_start} --study_stop={study_stop} --cut_points={cut_points} --controls_per_case={controls_per_case} --total_event_threshold={total_event_threshold} --episode_event_threshold={episode_event_threshold} --covariate_threshold={covariate_threshold} --age_spline={age_spline} --save_analysis_ready=TRUE --run_analysis=FALSE --df_output=model_output-{name}.csv"),
```

Comment:
Why have you put cox-ipw back a version?

Reply:
Because you updated the cox-ipw without telling me. Such change require to re-do all the stata section and re-run all the models on the server.

Reply:
You updated cox-ipw straight from v0.0.27 to v0.0.31 on February 28th 2024 (5 days after it was released): https://github.com/opensafely/post-covid-neurodegenerative/blob/9328c995f543d72205ea02c76673697be4828609/project.yaml

As far as I can tell, all models to date have been run using v0.0.31. Is there a particular reason you want to use v0.0.30 now?

Reply:
My mistake with the version. Although that does not explain why the released Stata models ran last month (as .csv). Anyway, regarding the  cox-ipw version, the vax parkinson hospitalised could not be run because the new version used .dta instead of .csv [Failed stata model](https://jobs.opensafely.org/investigating-events-following-sars-cov-2-infection/post-covid-neuro-v3/23406/42xh7ggmlsfi6tcj/) (same as the released stata models).